client/src/App.js

```diff
@@ -1,117 +1,31 @@
-                  </a>
-              </nav>
-          </div>
-
-          <div className="nav-scroller bg-white col-10 p-3 m-auto">
-              <h5>Deploy</h5>
-              <nav className="nav mb-1">
-                  <a className="m-1" href={"https://heroku.com/"}>
-                      <img alt={"Node.js"} src={"https://avatars1.githubusercontent.com/u/23211?s=48&v=4"} className={"rounded"}/>
-                  </a>
-                  <a className="m-1" href={"https://www.mongodb.com/cloud/atlas"}>
-                      <img alt={"expressjs"} src={"https://avatars1.githubusercontent.com/u/45120?s=48&v=4"} className={"rounded"}/>
-                  </a>
-              </nav>
-          </div>
-
-          <div className="nav-scroller bg-white col-10 p-3 m-auto table-responsive">
-              <h5>Contributors</h5>
-              <nav className="nav mb-1">
-                  <a className="m-1" href={"https://github.com/gayanvoice"}>
-                      <img alt={"gayanvoice"} src={"https://avatars1.githubusercontent.com/u/30500175?s=48&v=4"} className={"rounded"}/>
-                  </a>
-                  <a className="m-1" href={"https://github.com/srdjanRakic"}>
-                      <img alt={"srdjanRakic"} src={"https://avatars1.githubusercontent.com/u/16154503?s=48&v=4"} className={"rounded"}/>
-                  </a>
-                  <a className="m-1" href={"https://github.com/nikolalukovic"}>
-                      <img alt={"nikolalukovic"} src={"https://avatars0.githubusercontent.com/u/6897673?s=48&v=4"} className={"rounded"}/>
-                  </a>
-                  <a className="m-1" href={"https://github.com/Donghoon759"}>
-                      <img alt={"Donghoon759"} src={"https://avatars0.githubusercontent.com/u/32301380?s=48&v=4"} className={"rounded"}/>
-                  </a>
-                  <a className="m-1" href={"https://github.com/AWIXOR"}>
-                      <img alt={"AWIXOR"} src={"https://avatars0.githubusercontent.com/u/43700517?s=48&v=4"} className={"rounded"}/>
-                  </a>
-                  <a className="m-1" href={"https://github.com/ronaldaug"}>
-                      <img alt={"ronaldaug"} src={"https://avatars0.githubusercontent.com/u/33022876?s=48&v=4"} className={"rounded"}/>
-                  </a>
-                  <a className="m-1" href={"https://github.com/Ademking"}>
-                      <img alt={"Adem Kouki"} src={"https://avatars0.githubusercontent.com/u/12462188?s=48&v=4"} className={"rounded"}/>
-                  </a>
-
-              </nav>
-              <h6><span className="small">How to contribute? <a
-                  href={"https://github.com/gayanvoice/githubstats/blob/master/CONTRIBUTING.md"}> Add missing countries</a></span>
-              </h6>
-          </div>
-
-          <div className={'align-items-center p-3 mb-4 mt-3 text-center'}>
```

Comment:
You have removed the footer of the app which is used to display that this app githubstats.com is not affiliated with GitHub. I think it should be included in the app.

Reply:
I just make the Footer and Navbar as components to make your HTML code more visible !